app/repositories/application-repository.js

```diff
@@ -19,11 +19,11 @@ async function getByEmail (email) {
     })
 }
 
-async function getAll (limit, offset, sbi) {
+async function getAll (sbi, limit = 10, offset = 0) {
```

Comment:
Thinking about how this would be used, it might be better to have `limit` and `order` swapped for the reason that `offset` is more likely to be changed than `limit` which is possibly unlikely to be changed.

Reply:
move limit to last